src/config.nim

```diff
@@ -36,6 +36,8 @@ macro parseObject(obj: typed, key, val: string) =
 proc loadConfig*(filePath: string): Config =
   result = defaultConfig
   for rawLine in filePath.lines:
+    if rawLine.len == 0:
+      continue
     let line = rawLine.strip
     if line.len > 0 and line[0] == '#':
```

Comment:
```suggestion
    if (line.len > 0 and line[0] == '#') or line.len == 0:
```

Reply:
here, this is most optimal. i would drop `continue` entirely, but please, dont introduce another one + redundant if condition.

Reply:
I haven't seen anywhere in the if's any parentheses so i wanted to keep it consistent, plus the different if's for me show that they have different reasons to be there (one for empty lines and the other for comment lines)

Reply:
condition i suggest is already self-explanatory. and there are no parentheses.

Reply:
ok, after checking a little more, there are some cases with parentheses in if statements (in boomer.nim) but i think i still like it more this way. sometimes i prefer to lose some from "optimal" to gain in "clean"

Reply:
oh i just realized you edited it. ok i like it more with the new edit